.mergify.yml

```diff
@@ -1,4 +1,31 @@
 pull_request_rules:
+  - name: ask to fix commit message
+    conditions:
+      - -closed
+      - status-failure=Semantic Pull Request
+    actions:
+        comment:
+          message: |
+            Either the pull request or one of your commits does not follow our guidelines.
```

Comment:
Where will this message be shown? If they click on the link next to the mergify bot, they will see these details?